datasette/app.py

```diff
@@ -415,7 +433,24 @@ def metadata(self, key=None, database=None, table=None, fallback=True):
         assert not (
             database is None and table is not None
         ), "Cannot call metadata() with table= specified but not database="
-        databases = self._metadata.get("databases") or {}
+        metadata = {}
+
+        # TODO: change this to update_metadata, pass meta back allow the
+        # plugins to mutate it?
```

Comment:
Let's ditch the TODO comments before we merge the PR - they can live in PR comments the issue conversation instead.